resources/definitions/config_definitions.json

```diff
@@ -1942,6 +1942,20 @@
             "order": 2,
             "type": "integer"
         },
+        "dns.resolution_mode": {
+            "default": "os_default",
```

Comment:
`os`?

Reply:
yes, I renamed the options and I missed this :)